content/docs/user-guide/external-dependencies.md

```diff
@@ -110,6 +110,11 @@ $ dvc run -n download_file
 
 ### Local file system path
 
+For local paths outside of your project:
+
+> This includes different storage devices or partitions mounted on the same file
```

Comment:
I think it's more or less clear w/o that note

Reply:
Note removed.